Extension/package.json

```diff
@@ -5786,6 +5788,1589 @@
+                                "type": "boolean",
+                                "description": "%c_cpp.debuggers.cpplldb.externalConsole.description%",
+                                "default": false
+                            },
+                            "sourceFileMap": {
+                                "anyOf": [
+                                    {
+                                        "type": "object",
+                                        "description": "%c_cpp.debuggers.sourceFileMap.description%",
+                                        "default": {
+                                            "<source-path>": "<target-path>"
+                                        }
+                                    },
+                                    {
+                                        "description": "%c_cpp.debuggers.sourceFileMap.sourceFileMapEntry.description%",
+                                        "type": "object",
+                                        "default": {
+                                            "<source-path>": {
+                                                "editorPath": "",
+                                                "useForBreakpoints": true
+                                            }
+                                        },
+                                        "properties": {
+                                            "<source-path>": {
+                                                "type": "object",
+                                                "default": {
+                                                    "editorPath": "",
+                                                    "useForBreakpoints": true
+                                                },
+                                                "properties": {
+                                                    "editorPath": {
+                                                        "type": "string",
+                                                        "description": "%c_cpp.debuggers.sourceFileMap.sourceFileMapEntry.editorPath.description%",
+                                                        "default": ""
+                                                    },
+                                                    "useForBreakpoints": {
+                                                        "type": "boolean",
+                                                        "description": "%c_cpp.debuggers.sourceFileMap.sourceFileMapEntry.useForBreakpoints.description%",
+                                                        "default": true
+                                                    }
+                                                }
+                                            }
+                                        }
+                                    }
+                                ]
+                            },
+                            "logging": {
```

Comment:
The logging block should be removed, but I would recommend a way to get DAP logs to get information from the user to figure out what's going wrong in `lldb-dap`.

Reply:
You can set an environment variable `LLDBDAP_LOG` before running vscode and lldb-dap will dump out the log to the file you point to.

Reply:
The logging block does what it does for the other debug types - which as far as I can tell is very little (ie, emitting the debugger configuration to a log channel - but again, not specific to any debug type.

Reply:
there are other fields in logging too that would be no-op in the `lldb-dap` case. I would either remove it from the `package.json` definition so it would warn about it not existing.

Reply:
@WardenGnaw Should "either" be removed in your comment? I don't see an "or" mentioned.

Reply:
Removing.